Source/Controllers/DataExport/SPExportController.m

```diff
@@ -667,6 +669,8 @@ - (IBAction)changeExportOutputPath:(id)sender
                         @"class": classStr
                     };
 
+                    [MSACAnalytics trackEvent:@"chooseFileExportControllerError" withProperties: userInfo];
```

Comment:
ehhh, can we not do this yet? We are limited to number of events, so I would like to have generic events like `info` / `warning` / `error` and have all the rest in properties dictionary... to do that I'll implement logger object in Swift so we don't need to import MSAppCenter everywhere and rewrite half of the code next time when switching :)